packages/ember/tests/routing/basic_test.js

```diff
@@ -3386,3 +3386,45 @@ test("Exception during load of initial route is not swallowed", function() {
     bootApplication();
   }, /\bboom\b/);
 });
+
+test("Exception if outlet name is undefined in render", function() {
+  App.IndexRoute = Ember.Route.extend({
+    renderTemplate: function(controller, model) {
+      try {
```

Comment:
Can you use `throws` (see [here](http://api.qunitjs.com/throws/)) instead of the `try`/`catch`?